custom-connectors/SkyPointCloud/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "CLIENTID",
+          "scopes": [
+            "https://sicdpstage.onmicrosoft.com/api/user_impersonation openid profile offline_access"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "publisher": "SkyPoint Cloud",
+            "stackOwner": "SkyPoint Cloud"
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://sicdpstage.b2clogin.com/sicdpstage.onmicrosoft.com/b2c_1a_accountlink_signup_signin/oauth2/v2.0/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://sicdpstage.b2clogin.com/sicdpstage.onmicrosoft.com/b2c_1a_accountlink_signup_signin/oauth2/v2.0/token"
+            },
+            "refreshUrl": {
+              "value": "https://sicdpstage.b2clogin.com/sicdpstage.onmicrosoft.com/b2c_1a_accountlink_signup_signin/oauth2/v2.0/token"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#fefefe",
+    "capabilities": [],
+    "publisher": "SkyPoint Cloud"
+  }
```

Comment:
The `stackOwner` property appears to be missing. This can also be SkyPoint Cloud, so it could be:

`"stackOwner": "SkyPoint Cloud"`